CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## v0.7.1 (2021-11-12)
+
+* Changes to H26xParser, to make it more performatic. First extraction occurs using bytearrays instead of Bitstream. Performed benchmark resulted roughly in a reduction from 250s to 2.5s.
```

Comment:
Please undo the changelog change. I will generate the changelog automatically when I make a new release.

Reply:
Rolled back.